app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -881,6 +881,32 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
+    var useLocalFxAServer by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_use_local_fxa_server),
+        default = true
+    )
+
+    fun switchUseLocalFxAServer() {
+        val key = appContext.getPreferenceKey(R.string.pref_key_use_local_fxa_server)
+        val newValue = !preferences.getBoolean(key, true)
+        preferences.edit()
+            .putBoolean(key, newValue)
+            .apply()
+    }
+
+    var haveReadFxAAccountJson by booleanPreference(
```

Comment:
I don't think you need this nor `switchHaveReadFxAAccountJson` below (see my comment in FxaServer.kt)